data/human/free worlds 0 prologue.txt

```diff
@@ -15,51 +15,60 @@ mission "Pact Recon 0"
 	minor
 	name "Pirate Reconnaissance"
 	description "Fly through the <waypoints> systems, then report to <destination> so the Southern Defense Pact can analyze your logs of pirate activity (if any)."
-	waypoint "Alniyat"
-	waypoint "Han"
-	waypoint "Atria"
-	waypoint "Lesath"
 	source
-		attributes "south"
+		attributes "rim" "south"
 		not government "Pirate"
+		not system "Alniyat" "Atria" "Lesath" "Han"
 	destination "Glaze"
+	waypoint "Alniyat"
+	waypoint "Atria"
+	waypoint "Han"
+	waypoint "Lesath"
 	to offer
 		random < 60
 		not "event: war begins"
 	
 	on offer
 		conversation
-			`You can't help noticing that the freighter parked next to the <ship> is in very bad shape, its hull pockmarked with blaster scars, as well as larger dents that may be from missiles. The crew is busy welding new hull plating onto it; the captain is having an animated conversation with a young female militia officer.`
+			`You can't help noticing that the freighter parked next to the <ship> is in very bad shape, its hull pockmarked with blaster scars, as well as larger dents that may be from missiles. The crew and a number of men in militia uniforms are busy welding new hull plating onto it; the captain is having an animated conversation with a young female militia officer.`
```

Comment:
Naw.

Reply:
I believe warp's suggestion is technically considered correct, but it's one of those grammar rules I've traditionally been reluctant to agree with.

Right now, in this moment, personally I prefer warp's version with the comma, perhaps because as I get half way through, I briefly parse "conversation with a young female" which suggests rather a different emphasis of what's going on. (But I'm pretty ambivalent either way really.)